processor/src/main/java/com/linecorp/decaton/processor/runtime/Completion.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.decaton.processor.runtime;
+
+import java.util.concurrent.CompletionStage;
+
+import com.linecorp.decaton.processor.DeferredCompletion;
+
+/**
+ * An interface for completion status management of tasks.
+ */
+public interface Completion extends DeferredCompletion {
```

Comment:
Yeah, I thought about the same thing but decided to leave it because `DeferredCompletion` must be one of the interfaces that are referred often by user code, so even though we're going to bump major version, removing it will cause a lot of user applications to stop compiling.

I'm not sure what would be the best way here. I think keeping it is harmless in terms of maintainability, but if prefer to avoid leaving such garbage which has no goodenough reason to keep, we may mark it `@Deprecated` in this release and remove at some point in future?

Reply:
> DeferredCompletion must be one of the interfaces that are referred often by user code

Hm maybe.

Then sounds fine to keep the interface for now.